ui/DesignSystem/Primitive/Icon/TransactionState.svelte

```diff
@@ -20,20 +21,19 @@
   const radius = size / 2 - strokeWidth / 2;
   const circumference = 2 * Math.PI * radius;
 
+  const defaultDashLength = {
+    caution: progress || required / required,
+    positive: required,
+    negative: 0,
+  }[state];
+
   $: color = {
     caution: "var(--color-caution)",
     positive: "var(--color-positive)",
     negative: "var(--color-negative)",
   }[state];
 
-  const defaultDashLength = {
-    caution: progress || 100 / 6,
-    positive: 100,
-    negative: 0,
-  };
-
-  $: dashLength =
-    progress === 0 ? 100 / 6 : progress || defaultDashLength[state];
+  $: dashLength = progress === 0 ? defaultDashLength : progress;
```

Comment:
The mixed states are not shown properly anymore, I believe because of this change. It should be possible to be in the `negative` variant (red) but with a rotating dash.

Reply:
Addressed in 69b77fc